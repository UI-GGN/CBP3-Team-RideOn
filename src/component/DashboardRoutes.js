import React from "react";
import DashboardBreadCrump from "./DashboardBreadCrump";

function DashboardRoutes() {
  return (
      <DashboardBreadCrump type="Routes"/>
  );
}

export default DashboardRoutes;
