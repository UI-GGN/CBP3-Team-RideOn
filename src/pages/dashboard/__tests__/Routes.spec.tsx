import {render} from "@testing-library/react";
import React from "react";
import DashboardRoutes from "../Routes";

describe("Routes test", () => {
  test("renders the component", async () => {
    const {getByText} = render(<DashboardRoutes />);

    expect(getByText("Dashboard")).toBeVisible();
    expect(getByText("Routes")).toBeVisible();
  });
  it("renders correctly", () => {
    const {baseElement} = render(<DashboardRoutes />);
    expect(baseElement).toMatchSnapshot();
  });
});
