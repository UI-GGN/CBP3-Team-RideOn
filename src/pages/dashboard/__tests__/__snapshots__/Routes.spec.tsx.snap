// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes test renders correctly 1`] = `
<body>
  <div>
    <div
      class="breadcrumb"
    >
      <div>
        <nav
          aria-label="breadcrumbs"
          class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root css-1j6ngzq-MuiTypography-root-MuiBreadcrumbs-root"
        >
          <ol
            class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
          >
            <li
              class="MuiBreadcrumbs-li"
            >
              <p
                align="Center"
                class="MuiTypography-root MuiTypography-body1 css-1n8z4pj-MuiTypography-root"
                font="Roboto"
              >
                Dashboard
              </p>
            </li>
            <li
              aria-hidden="true"
              class="MuiBreadcrumbs-separator css-1wuw8dw-MuiBreadcrumbs-separator"
            >
              /
            </li>
            <li
              class="MuiBreadcrumbs-li"
            >
              <p
                align="Center"
                class="MuiTypography-root MuiTypography-body1 css-136mgio-MuiTypography-root"
                font="Roboto"
              >
                Routes
              </p>
            </li>
          </ol>
        </nav>
      </div>
    </div>
  </div>
</body>
`;
